### ISSUE TITLE
Intermitent 403 in cran checks?

### ISSUE THREAD

**Issue by dblodgett-usgs** [hydroshare/hydroshare#4582]
**Description of the bug**
In a CRAN package I maintain, a link to a hydroshare resource gets flagged as returning a 403 (Forbidden) intermittently. I have no way of reproducing this but am wondering if there is a more stable URL that a link checker will always get a 200 response from? 

The resource URL in question: https://www.hydroshare.org/resource/6092c8a62fac45be97a09bfd0b0bf726/

CRAN check with failure: https://win-builder.r-project.org/incoming_pretest/nhdplusTools_0.5.2_20220407_183527/Debian/00check.log

**Comment by devincowan** [comment]
@dblodgett-usgs have you still been experiencing this intermittent issue?

**Comment by dblodgett-usgs** [comment]
I don't believe so. My CRAN check scheme has evolved to avoid links that aren't 100%.
